js/scheduler.js

```diff
@@ -515,7 +515,7 @@
 						item.find('label').removeClass('active');
 						temp = recur.BYDAY.split(',');
 						for (i = 0, l = temp.length; i < l; i++) {
-							item.find('input[data-value="' + temp[i] + '"]').parent().addClass('active');
+							item.find('input[data-value="' + temp[i] + '"]').attr('checked',true).parent().addClass('active');
```

Comment:
I believe this should be `.prop('checked', true)` not `attr('checked', true)`. http://api.jquery.com/prop/

    the most important concept to remember about the checked attribute is that it does not correspond to the checked property. The attribute actually corresponds to the defaultChecked property and should be used only to set the initial value of the checkbox. The checked attribute value does not change with the state of the checkbox, while the checked property does.